.editorconfig

```diff
@@ -5,14 +5,18 @@ root = true
 
 # Don't use tabs for indentation.
 [*]
-end_of_line = lf
 indent_style = space
+tab_width = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+guidelines = 120
```

Comment:
⚠️ fine to move, but please remove the duplicate at the bottom

Reply:
Nice -- didn't see that.